tests/query/allocation_policies/test_bytes_scanned_rejecting_policy.py

```diff
@@ -107,53 +107,6 @@ def test_consume_quota(
     assert allowance.can_run
 
 
-@pytest.mark.redis_db
```

Comment:
why are you removing this test?

Reply:
it is covered by `test_does_not_throttle_and_then_throttles` and `test_does_not_throttle_and_then_throttles` has more realistic numbers